OpenSim/Common/Component.h

```diff
@@ -755,19 +755,25 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
      * lets you get information about the connection (like if the connector is
      * connected), but does not give you access to the connector's connectee.
      * For that, use getConnectee().
+     *
+     * Example:
```

Comment:
`Example:` -> `<b>C++ example</b>` (as done [here](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Simulation/SimbodyEngine/Joint.h#L62), for example) because the doxygen serves more than just C++ users.